nginx/nginx.go

```diff
@@ -236,6 +238,15 @@ func (lb *nginxLoadBalancer) Update(entries controller.IngressUpdate) error {
 
 func (lb *nginxLoadBalancer) update(entries controller.IngressUpdate) (bool, error) {
 	log.Debugf("Updating loadbalancer %s", entries)
+
+	if lb.Conf.AccessLog {
+		log.Infof("Access log directive enabled. Log files will be written in: %s", lb.Conf.AccessLogDir)
+		if err := os.MkdirAll(lb.Conf.AccessLogDir, 0755); err != nil {
+			log.Errorf("Could not create access log dir. Error: %v", err)
+			return false, err
+		}
+	}
```

Comment:
Is this necessary? Can't we just rely on the dockerfile or OS level to create the directory? This also means the feed-ingress process needs write permission in the parent directory.